selfie.dev/src/pages/py/cache.mdx

```diff
@@ -34,33 +35,35 @@ You control whether a given value is being read or written using the `_TODO`, `/
 
-The hazard is that the cached result *is not testing the function call anymore*. It is just a convenient way to generate sample data for testing *other* parts of the system.
+The hazard is that the cached result _is not testing the function call anymore_. It is just a convenient way to generate sample data for testing _other_ parts of the system.
 
-```java
+```python
 var brittleAssumption = cacheSelfie(() -> expensiveOperation()).toBe("sand")
 buildStuffOn(brittleAssumption)
 ```
 
-Perhaps the `toBe` snapshot was recorded a year ago, and the `expensiveOperation` has changed since then. Perhaps someone manually edited the recorded snapshot, and `expensiveOperation` has never returned a value anything like the snapshot. ***The function being cached is not being tested.***
+Perhaps the `toBe` snapshot was recorded a year ago, and the `expensiveOperation` has changed since then. Perhaps someone manually edited the recorded snapshot, and `expensiveOperation` has never returned a value anything like the snapshot. **_The function being cached is not being tested._**
 
-If you have a test with multiple `cacheSelfie` calls, avoid using `_TODO`. You can have a situation where you recorded the ending of a test, and then later changed the beginning with `_TODO`. The ending won't update itself automatically ([yet](https://github.com/diffplug/selfie/issues/237)), so you might cache an inconsistent state. You can avoid this problem by only using `//selfieonce` and `//SELFIEWRITE`.
+If you have a test with multiple `cacheSelfie` calls, avoid using `_TODO`. You can have a situation where you recorded the ending of a test, and then later changed the beginning with `_TODO`. The ending won't update itself automatically, so you might cache an inconsistent state. You can avoid this problem by only using `//selfieonce` and `//SELFIEWRITE`.
 
 ## Strings and binary
 
 You have these choices for specifying the data in a snapshot:
 
-```java
+```python
 cacheSelfie(() -> "string").toBe("string")
 cacheSelfie(() -> "string").toMatchDisk()
-cacheSelfieBinary(() -> new byte[3]).toBeBase64("AAAA")
-cacheSelfieBinary(() -> new byte[3]).toBeFile("pkg/someFile.ext")
-cacheSelfieBinary(() -> new byte[3]).toMatchDisk()
+cacheSelfieBinary(() -> bytearray[3]).toBeBase64("AAAA")
+cacheSelfieBinary(() -> bytearray[3]).toBeFile("pkg/someFile.ext")
+cacheSelfieBinary(() -> bytearray[3]).toMatchDisk()
 ```
 
 The `toMatchDisk` method is nice because Selfie will garbage-collect the snapshot if it isn't being used, you can "set it and forget it". `toBe` is nice to read inline with the code, and `toBeFile` is nice because you can open the result in external programs.
 
 ## Roundtripping typed data
 
+{/* Python doesn't have Roundtrip, make an implementation? */}
```

Comment:
Are we implementing our own Roundtrip for python? I left it as is for now and put a note to change.

Reply:
I would assume that people will see these docs before we have finished implementing everything. It's safer to say
`**TODO: we don't have Roundtrip in the Python, are we going to implement this?**` than to hide it behind a comment that might not get seen. A great thing about opensource versus commercial is that with opensource, you can advertise your problems and someone might come and fix it for us!